src/main/groovy/5addSeedJob.groovy

```diff
@@ -0,0 +1,52 @@
+import java.util.logging.Logger
+import jenkins.*
+import jenkins.model.*
+import hudson.model.*
+
+@Grapes([
+    @Grab(group='org.yaml', module='snakeyaml', version='1.17')
+])
+import org.yaml.snakeyaml.Yaml
+
+Logger logger = Logger.getLogger("")
+Jenkins jenkins = Jenkins.getInstance()
+Yaml yaml = new Yaml()
+
+String configPath = System.getenv("JENKINS_CONFIG_PATH")
+String configText = ''
+try {
+    configText = new File("${configPath}/seed_config.yml").text
+} catch (FileNotFoundException e) {
+    logger.severe("Cannot find config file path @ ${configPath}/see_config.yml")
+    jenkins.doSafeExit(null)
+    System.exit(1)
+}
+
+Map seedConfig = yaml.load(configText)
+seedJobName = seedConfig.NAME
+seedXmlFilePath = seedConfig.XML_PATH
+
+if (!seedJobName || !seedXmlFilePath) {
+    logger.severe("Missing data for seed job. Please ensure the " +
+                  "seed_config.yml file has a NAME and XML_PATH")
+    jenkins.doSafeExit(null)
+    System.exit(1)
+}
+
+try {
+    seedXml = new File("${seedXmlFilePath}")
+} catch (FileNotFoundException e) {
+    logger.severe("Cannot find config file path @ ${seedXmlFilePath}")
+    jenkins.doSafeExit(null)
+    System.exit(1)
+}
+
+existingSeedJob = jenkins.getJob(seedJobName)
+if (existingSeedJob) {
+    logger.info("Jobs by the name: ${seedJobName} exists. Deleting.")
+    existingSeedJob.delete()
+}
+
+xmlByteStream = new ByteArrayInputStream(seedXml.getBytes())
+jenkins.createProjectFromXML(seedJobName, xmlByteStream)
```

Comment:
I think this could throw errors if there is malformed xml. Can you add a try/catch if so?